python/tests/test_datetime.py

```diff
@@ -72,6 +72,23 @@ def test_construct_0d_datetime_mismatch(unit1, unit2):
         sc.Variable(value=np.datetime64('now', unit1), unit=unit2)
 
 
+def test_construct_0d_datetime_nounit():
+    # Can make a datetime variable without unit but cannot do anything
+    # with it except set its unit.
```

Comment:
Can't we slice, copy, add, ...? Not saying you should test it here, but the comment soundsmisleading?

Reply:
- copy: Yes
- slice: Yes, but the slice inherits the limitations
- subtract: Yes, when both operands have dtype=datetime64 and unit=one. But this means that both are uninitialised.

But none of those sound meaningful to me